packages/mux-video/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.2.0",
   "type": "commonjs",
   "description": "A custom mux video element for the browser that Just Works™",
+  "module": "dist/index.mjs",
```

Comment:
a little concerned about this causing conflicts for anyone trying to use the web component in SSR envs. Gonna move forward for now but I think we'll need to do some smoke testing before cutting a release.

Reply:
May be fine since it's `"module"` not `"main"` fwiw